paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/MainActivity.kt

```diff
@@ -61,7 +62,16 @@ class MainActivity : AppCompatActivity() {
             MenuItem(
                 titleResId = R.string.paymentsheet_serverside_confirmation_title,
                 subtitleResId = R.string.paymentsheet_serverside_confirmation_subtitle,
-                klass = ServerSideConfirmationActivity::class.java,
+                klass = ServerSideConfirmationCompleteFlowActivity::class.java,
+                badge = MenuItem.Badge(
+                    labelResId = R.string.beta_badge_label,
+                    onClick = this::openDecouplingBetaLink,
+                ),
+            ),
+            MenuItem(
+                titleResId = R.string.flow_controller_serverside_confirmation_title,
+                subtitleResId = R.string.paymentsheet_serverside_confirmation_subtitle,
+                klass = ServerSideConfirmationCustomFlowActivity::class.java,
```

Comment:
I think we should update the list with clearer naming and separation. Happy to do this as a follow-up.